WooCommerce/Classes/POS/Presentation/Reusable Views/POSSendReceiptView.swift

```diff
@@ -57,10 +77,16 @@ struct POSSendReceiptView: View {
             .background(Color.posOverlayFillInverted)
             .cornerRadius(Constants.buttonCornerRadius)
             .contentShape(Rectangle())
+            .disabled(isLoading)
 
             Spacer()
         }
         .padding()
+        .onChange(of: textFieldInput) { _ in
+            if errorMessage != nil {
+                errorMessage = nil
```

Comment:
Maybe we can just call `errorMessage = nil` here, no need for the nil-check 🤔

Reply:
Removed on https://github.com/woocommerce/woocommerce-ios/pull/14651/commits/3250b1fd8b2e08845c1c5479f94ffd4d9a255e53